core/chains/evm/config/toml/defaults.go

```diff
@@ -65,6 +74,64 @@ func init() {
 	slices.SortFunc(DefaultIDs, func(a, b *big.Big) int {
 		return a.Cmp(b)
 	})
+
+	// read the custom defaults overrides
+	dir := os.Getenv(string(env.CustomDefaultsEnvKey))
+	if dir == "" {
+		// short-circuit; no default overrides provided
+		return
+	}
+
+	// Read directory contents
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		fmt.Println("Error reading directory:", err)
```

Comment:
More context to this log would be useful, e.g maybe `config/toml/defaults.go: Error reading custom defaults directory:`

Reply:
Similar comments on the rest of the fmt.Println statements in this func

Reply:
added, thanks for the call out